egui_demo_lib/src/apps/demo/misc_demo_window.rs

```diff
@@ -374,7 +393,13 @@ impl Tree {
             .into_iter()
             .enumerate()
             .filter_map(|(i, mut tree)| {
-                if tree.ui_impl(ui, depth + 1, &format!("child #{}", i)) == Action::Keep {
+                if tree.ui(
+                    ui,
+                    depth + 1,
+                    &format!("child[{}][{}]", depth, i),
```

Comment:
This naming scheme leads to name clashes:

<img width="166" alt="Screen Shot 2021-08-16 at 21 35 49" src="https://user-images.githubusercontent.com/1148717/129619784-d24e049c-5fff-444a-81c4-a8cc68a536b6.png">

A robust alternative is to give the full path to each child, e.g. `root/0/2/3`, with `&format("{}/{}", parent_name, i)`

Reply:
Ah! I didn't noticed it - my bad - I'll change it